packages/secure_storage/amplify_secure_storage_dart/lib/src/amplify_secure_storage_dart.dart

```diff
@@ -64,6 +64,10 @@ class AmplifySecureStorageDart extends AmplifySecureStorageInterface
       );
     };
   }
+
+  // Required to register the plugin on windows and linux platforms.
+  // https://docs.flutter.dev/packages-and-plugins/developing-packages#plugin-platforms
+  static void registerWith() {}
```

Comment:
Can this can be a no-op? There isn't a instance from the current impl to map here.